manual-utatane/manual-en.md

```diff
@@ -2,114 +2,109 @@
+When the reveal step is finished for a girl having the "Cool" Orientation, the player with control declares a target girl to increase the Favor level with by 3. If other players have more hidden Control Points over the girl, players may interrupt to reveal their Control Points to change the target girl.
 
-When all Control point reveal step ends, advanced to 1st turn's Action phase. Game will progress same as basic game after this phase.
+When all Control Point reveal steps have ended, advance to the 1st turn Action Phase. The game will progress in the same way as the basic game after this phase.
 
 ![Orientation](img/seikou-en.png)
 
-### Changes about Support Point
+### Changes to Support Points
 
-Additional Support point will be modified below in a game with Orientation.
+Additional Support Points will be added in a game with Orientation.
 
-- +3 additional Support point after 3rd turn (+1 -> +3)
-- +5 additional Support point after 6th turn (+3 -> +5)
+- +3 additional Support Points after the 3rd turn (+1 -> +3)
+- +5 additional Support Points after the 6th turn (+3 -> +5)
 
-Important change from basic game: **Assigning Support point to a pair multiple time in Setup is forbidden.** It only restricts assignment in Setup phase. **Players can assign additional Support point in any combination, including duplicated assignment.**
+Important change from the basic game: **Assigning Support Points to a pair multiple times in setup is forbidden.** This only restricts assignment in the Setup Phase. **Players can assign additional Support Points in any combination, including duplicated assignment.**
 
 ```plaintext
-At the Setup: you cannot assign like "5 to Akane-Shirakaba, 4 to Akane-Shirakaba, 3 to Midorino-Tsuge, 2 to Midorino-Sorai, 1 to Murafuji-Tsuge" (Akane-Shirakaba is duplicated)
-
-When you assign "5 to Akane-Shirakaba, 4 to Sorai-Shirakaba, 3 to Midorino-Tsuge, 2 to Midorino-Sorai, 1 to Murafuji-Tsuge" at Setup, you can assign "2 to Sorai-Shirakaba, 1 to Midorino-Tsuge" at the end of the 3rd turn.
+During setup: you cannot assign "5 to Akane-Shirakaba, 4 to Akane-Shirakaba, 3 to Midorino-Tsuge, 2 to Midorino-Sorai, 1 to Murafuji-Tsuge" (Akane-Shirakaba is duplicated)
 
+When you assign "5 to Akane-Shirakaba, 4 to Sorai-Shirakaba, 3 to Midorino-Tsuge, 2 to Midorino-Sorai, 1 to Murafuji-Tsuge" during setup, you can assign "2 to Sorai-Shirakaba, 1 to Midorino-Tsuge" at the end of the 3rd turn.
 ```
 
-At the end of the game, the player assigned Support point most to the Fatal couple / Yuri-Polygamy group.
-(Additional Support point are also summarized.)
+At the end of the game, the player who assigned the most Support Points to the Fated Couple / Yuri Polygamy group is the winner. Additional Support Points are also included in winner determination.
```

Comment:
You suggested "Fated couple" instead to "Fatal couple". I am willing to adopt your suggestion, but I cannot understand defferences between them. I am glad if you tell me the reason.

Reply:
Fated and fatal can be interpreted in the same way. But it's much more common for fatal to refer to something dangerous, or deadly.

You could also say "Fateful Couple", although I don't think it's quite as clear. "Fated Couple" implies the couple was "meant to be".